schema/dcschema.mcf

```diff
@@ -2226,3 +2226,8 @@ name: "MedicalTest"
 typeOf: schema:Class
 subClassOf: dcs:MedicalProcedure
 
+Node: dcid:demElevation
+typeOf: schema:Property
+name: "demElevation"
+descriptionUrl: "http://download.geonames.org/export/dump/readme.txt"
+description: "Average elevation based on Digital Elevation Model (DEM), in meters, integer."
```

Comment:
Should the DEM model, "average", and "unit" be part of the measurement or the variable, instead of property? Maybe just use https://schema.org/elevation instead?

Reply:
This is actually different from "elevation", see here http://download.geonames.org/export/dump/readme.txt  . Search for "elevation" and "dem". Seems like 'average' is the definition of this property?
I agree unit (meters) should go into observation -- changed.